src/main/webapp/user_profile.jsp

```diff
@@ -42,7 +42,7 @@
         <div id="container">
             <button class="help-button" id="need-help-button" onclick="showNeedHelp()">Need help</button>
             <button class="help-button" id="offer-help-button" onclick="showOfferHelp()">Offer help</button>
-            <i class="fa fa-plus-circle fa-3x" id="create" onclick="showModal()"></i>
+            <i class="fa fa-plus-circle fa-3x" id="create-task-button" onclick="showModal()"></i>
```

Comment:
Is the name of the onclick function incorrect? I think the function has been renamed to `showCreateTaskModal` right?

Reply:
I renamed the function on the homepage script file but not the user_profile script :)

Reply:
Ah my fault. I thought it was the index.jsp 🤣